src/migrations/2019_02_14_000001_update_tax_areas_table.php

```diff
@@ -121,6 +125,8 @@ public function down()
                     ]);
             });
 
+        DB::table($this->tax_areas)->whereNull('name')->delete();
+
```

Comment:
Updating old migration????

Reply:
was the only way I could find to get the new migrations to run, don't like it tho